iosApp/Configuration/Config.xcconfig

```diff
@@ -1,3 +1,3 @@
 TEAM_ID=
-BUNDLE_ID=com.myapplication.MyApplication
-APP_NAME=My application
+BUNDLE_ID=com.sciack.knews
```

Comment:
android applicationId is such:
com.github.sciack.news
is it intentional

Reply:
forgot `github` :)